contracts/interfaces/IValidatorSet.sol

```diff
@@ -1,22 +1,23 @@
-pragma solidity 0.4.25;
-
+pragma solidity 0.5.2;
+import "./IRandom.sol";
+import "./IBlockReward.sol";
 
 interface IValidatorSet {
-    function initialize(address, address, address, address[], uint256, uint256) external;
+    function newValidatorSet() external;
+    function initialize(address, address, address, address[] calldata, uint256, uint256) external;
     function blockRewardContract() external view returns(address);
     function changeRequestCount() external view returns(uint256);
     function erc20TokenContract() external view returns(address);
-    function getPendingValidators() external view returns(address[]);
-    function getPreviousValidators() external view returns(address[]);
-    function getValidators() external view returns(address[]);
+    function getPendingValidators() external view returns(address[] memory);
+    function getPreviousValidators() external view returns(address[] memory);
+    function getValidators() external view returns(address[] memory);
     function isReportValidatorValid(address) external view returns(bool);
     function isValidator(address) external view returns(bool);
     function isValidatorOnPreviousEpoch(address) external view returns(bool);
-    function MAX_VALIDATORS() external pure returns(uint256);
-    function newValidatorSet() external;
-    function poolStakers(address) external view returns(address[]);
+    function maxValidators() external pure returns(uint256);
```

Comment:
We need to keep `MAX_VALIDATORS` because we don't have `maxValidators` getter in `ValidatorSet` contract.

Constants in Solidity should be capitalized: https://solidity.readthedocs.io/en/v0.5.2/style-guide.html#constants